core/src/Streamly/Internal/Data/Stream/Expand.hs

```diff
@@ -37,6 +37,9 @@ module Streamly.Internal.Data.Stream.Expand
     -- ** Append
     , append2
 
+    -- ** Modifier for Stream fusion
+    , fuse
```

Comment:
This is a common operation, should not be in the Expand module. Can go in the Stream/Type module.